implementations/rust/ockam/ockam_command/src/args.rs

```diff
@@ -0,0 +1,93 @@
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Clone, Debug, Parser)]
+#[clap(name = "ockam", version)]
+pub struct CliArgs {
+    #[clap(subcommand)]
+    pub command: Command,
+    /// Increase verbosity of logging output.
+    ///
+    ///   `-v` => Info level, and output extra debug information.
+    ///   `-vv` => Debug level.
+    ///   `-vvv` => Trace level.
+    #[clap(long, short, parse(from_occurrences))]
+    pub verbose: u8,
+}
+
+#[derive(Clone, Debug, Args)]
+pub struct InletOpts {
+    /// Ockam's cloud node address
+    pub cloud_addr: String,
```

Comment:
Are we using default saved Identity for SecureChannel while creating inlet/outlet? Did you consider using specifying identity_id here? At least optionally.

Reply:
I think we probably will want this later though, but for now it's fine.